### DISCUSSION TITLE
MacOS: Packet sender not working after migration to MacOS Monterey

### DISCUSSION THREAD

**Discussion by GiovanniCastello** [dannagle/PacketSender#275] [Q&A]
Hi, I am experiencing issues with Packet sender after migration to latest version of MacOS.

I cannot enable either UDP either TCP server within PacketSender.

On startup I get popup mjessage:
Packet Sender attempted (and failed) to bind to port 20000.
 - Are you running multiple instances? 
 - Trying to bind to a missing custom IP?

Thanks for help in advance.

**Comment by dannagle** [comment]
Not sure what could've happened. Perhaps try clearing your settings and trying again?
The settings file is stored here:
`/Users/YOUR_USERNAME/Library/Application Support/PacketSender/ps_settings.ini`

You can make a backup first.
